### PR TITLE
add brion module to be able to compose it with bluepy and read *.bbp reports

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

**Comment by matz-e** [comment]
OK to test

File: deploy/packages/bbp-packages.yaml [hunk 1]
```diff
@@ -55,6 +55,7 @@ packages:
       - touchdetector@4.4.2
       - touchdetector@5.0.1
       - touchdetector@5.1.0
+      - brion@3.0.0
```
Comment (matz-e) [inline deploy/packages/bbp-packages.yaml]: Please also whitelist `brion` in `deploy/configs/applications/modules.yaml`

**Review by matz-e — CHANGES_REQUESTED** [review]

File: deploy/packages/bbp-packages.yaml (hunk 1 above)
Comment (matz-e) [reply deploy/packages/bbp-packages.yaml]: From our [README](https://github.com/BlueBrain/spack#managing-centrally-built-modules-and-packages):
> To actually enable the generation of corresponding module files, new software has to be whitelisted in the module configuration for spack, e.g. by editing:
>
>     $ vim spack/deploy/configs/applications/modules.yaml
>
> Look for a key whitelist, and add the package. Packages listed in the module whitelist should be as generic as possible, i.e., not tied to specific versions.

**Review by matz-e — APPROVED** [review]

File: deploy/packages/bbp-packages.yaml (hunk 1 above)
Comment (genric) [reply deploy/packages/bbp-packages.yaml]: Fell into the same trap again :) sorry, and thanks for reminding about the updated readme!

**Comment by pramodk** [comment]
module testing was failing with:

```
+ . /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/455/config/modules.sh
/jenkins/06/workspace/hpc.module-testing@tmp/durable-f6da3d42/script.sh: line 5: /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/455/config/modules.sh: No such file or directory
script returned exit code 1
```

@matz-e  : not sure why `config/modules.sh` was not generated [in here](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.module-testing/detail/hpc.module-testing/392/pipeline/).

@genric : I have added you to this repo (and Jenkins CI). So you can directly push to this repo instead of fork.

**Comment by matz-e** [comment]
> @genric : I have added you to this repo

@pramodk note that I added NSE as a team to this repo already. I think only the Jenkins part was needed? We should stop adding single users here.

**Comment by pramodk** [comment]
>  note that I added NSE as a team to this repo already. I think only the Jenkins part was needed? We should stop adding single users here.

Ahh missed that. Didn't know there was a team. For Jenkins part I [created ticket](https://bbpteam.epfl.ch/project/issues/browse/HELP-10890).

>  : not sure why config/modules.sh was not generated in here.

I missed that the `modules.sh` was not generated because [builds are failing](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.spack-deployment/detail/hpc.spack-deployment/1578/tests/).

**Comment by genric** [comment]
Any ideas why build is failing?

**Comment by matz-e** [comment]
I think we need another rebase, close and reopen of the PR here :-/

**Comment by genric** [comment]
I don't see PYTHONPATH being extended when I load the module. What kind of magic needs to be done so that python bindings are available?

**Comment by matz-e** [comment]
> I don't see PYTHONPATH being extended when I load the module. What kind of magic needs to be done so that python bindings are available?

Have a look at the `mvdtool` recipe! I don't think Brion does that?

**Pull request merged**